### PR TITLE
Localization handback

### PR DESCRIPTION
**Pull request by nguerrera**
Replacement for #2407, parented by the right branch.

cc @khoiph1

### CODE REVIEW

File: src/Tasks/Common/Resources/xlf/Strings.de.xlf [hunk 1]
```diff
@@ -1,354 +1,354 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="de" original="../Strings.resx">
+  <file datatype="xml" source-language="en" target-language="de-DE" original="../Strings.resx">
```
Comment (nguerrera) [inline src/Tasks/Common/Resources/xlf/Strings.de.xlf]: @khoiph1 I noticed in several of these that target-language is changed to a more specific culture than we actually use it as in the product. I would prefer if these continued to match the file name.

File: src/Tasks/Common/Resources/xlf/Strings.cs.xlf [hunk 2]
```diff
@@ -1,354 +1,354 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (nguerrera) [inline src/Tasks/Common/Resources/xlf/Strings.cs.xlf]: UTF-8 BOM removed. It is not required so I'll let that go, but noting the diff.

Comment (nguerrera) [reply src/Tasks/Common/Resources/xlf/Strings.cs.xlf]: If possible, consider changing the tooling that generated this to leave the BOM/NO-BOM state intact.

File: src/Tasks/Common/Resources/xlf/Strings.de.xlf (hunk 1 above)
Comment (nguerrera) [reply src/Tasks/Common/Resources/xlf/Strings.de.xlf]: I'm going to revert this for this insertion. I'm 99% sure nothing in the product is using target-language, but I don't want to take a change other than translation at the end of the release. We can talk about it on the next handback.

**Review by peterhuene — APPROVED** [review]

**Pull request merged**